ssqatest/tests/backend/misc_tests/test_create_new_product.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+from ssqatest.src.api_helpers.ProductsAPIHelper import ProductsAPIHelper
+from ssqatest.src.dao.products_dao import ProductsDAO
+from ssqatest.src.utilities.genericUtilities import generate_random_string
+import logging as log
+pytestmark= [pytest.mark.backend]
+
+"""
+    This is a test to create a product using API post call and then asserting it was created.
+"""
+
+@pytest.mark.be001
+#create a payload and run a post API call to create a product
+def test_create_a_product():
+    payload=dict()
+    payload["name"]=generate_random_string(5,"lp")
+    payload["weight"]= "2"
```

Comment:
Looks a little compact. add some spaces. Try to figure out what the pep-8 convention is for spaces. lookit up or ask ChatGPT.
I am talking about space between lines. Like post this in chatGPT and ask it what would be the right format to follow pep-8 convention.